scipy/stats/_continuous_distns.py

```diff
@@ -9008,16 +9008,16 @@ def _argcheck(self, beta, m):
 
 def _argus_phi(chi):
     """
-    Utility function for the argus distribution
-    used in the pdf, sf and moment calculation.
-    For small chi, use gammaincinv for better precision as for all x > 0:
+    Utility function for the argus distribution used in the pdf, sf and
+    moment calculation.
+    For small chi, use gammainc for better precision as for all x > 0:
```

Comment:
I point out a few lines below that we always use gammainc for better speed. But let's see if we get consistent timing results.

Reply:
I see. I meant that introducing it with "For small chi" can be interpreted to mean that we'll only use `gammainc` for small chi and something else for larger chi. After all, this is what "For small chi" meant before the most recent commit. What we mean is that "We use `gammainc` instead of the usual definition because it is more precise for small chi".